styles/style.css

```
@@ -0,0 +1,517 @@
+  }
+  .empty-block {
+    width: 0;
+  }
+  .section--reviews__column {
+    display: flex;
+    flex-direction: row;
+  }
+}
+
+.navigation {
+  width: 100%;
+  border-bottom: 3px solid #ecd19e;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 40px;
+}
+.navigation--title {
+  float: left;
+  display: flex;
+  justify-content: center;
+}
+.navigation--links {
+  float: right;
+}
+nav {
+  display: flex;
+  justify-content: flex-start;
+  font-family: "Raleway", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  flex-wrap: wrap;
+}
+nav a {
+  text-decoration: none;
+  color: #1e150f;
+  margin-right: 55px;
+}
+
+nav a:last-child {
+  margin-right: 0;
+}
+
+main {
+  display: grid;
+  row-gap: 50px;
+  width: 100%;
+  background-color: #ffffff;
+}
+.about {
+  display: flex;
+  justify-content: start;
+  background-image: url("../images/main/section-1/bottle-of-serum.png");
+  background-position: left top;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.section--about__description {
+  display: grid;
+  justify-items: end;
+  row-gap: 25px;
+  padding-top: 50px;
+}
+
+.section--about__subtitle h5 {
+  position: relative;
+  font-family: "Raleway", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 23px;
+  line-height: 28px;
+  padding-bottom: 15px;
+}
+.section--about__subtitle h5::after {
+  content: "";
+  position: absolute;
+  left: 50%;
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  background-color: #e9ca91;
+  transform: translateX(-50%);
+}
+
+.section--about__properties {
+  width: 70%;
+  display: flex;
+  flex-direction: row;
+  font-family: "Open Sans", sans-serif;
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  margin: 0 auto;
+}
+.section--about__properties-2 {
+  border-left: 1px solid #000000;
+  border-right: 1px solid #000000;
+}
+
+.section--about__text {
+  font-family: "Open Sans", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 22px;
+}
+
+.section--about__button a {
+  background-color: rgb(30, 21, 15);
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+  text-align: center;
+  font-size: 15px;
+  font-family: "Open Sans", sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 50px;
+  color: rgb(255, 255, 255);
+  transition: all 0.3s;
+}
+
+.section--about__button a:hover {
+  background-color: #cecece;
+}
+
+.specification {
+  margin: 0 auto;
+}
+
+.section--specification {
+  width: 70%;
+  margin: 0 auto;
+}
+.section--specification__title {
+  margin-bottom: 40px;
+}
+.section--specification__title h3 {
+  text-align: center;
+  position: relative;
+  font-family: "Raleway", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 60px;
+  line-height: 70px;
```

```suggestion
  line-height: 1;
  padding: 0 0 30px;
```

here and in other similar places